python/paddle/fluid/layers/nn.py

```diff
@@ -9654,9 +9654,16 @@ def space_to_depth(x, blocksize, name=None):
         .. code-block:: python
 
             data = fluid.layers.data(
-                name='data', shape=[1, 4, 2, 2], dtype='float32')
+                name='data', shape=[1, 4, 2, 2], dtype='float32', append_batch_size=False)
             space_to_depthed = fluid.layers.space_to_depth(
                 x=data, blocksize=2)
+
+            exe = fluid.Executor(fluid.CUDAPlace(0))
+            data_np = np.arange(0,16).reshape((1,4,2,2)).astype('float32')
+            out = exe.run(fluid.default_startup_program())
```

Comment:
`space_to_depth` doesn't have a parameter, so line9663 is unnecessary.

Reply:
done, thx